apps/coscrad-frontend/src/components/resources/photographs/photograph-detail.full-view.presenter.tsx

```diff
@@ -34,8 +37,14 @@ export const PhotographDetailFullViewPresenter = ({
                 )}
             />
             <ImageFullPageWidth image={image} />
-            <ResourceNamePresenter name={name} variant="h2" />
+            <ResourceDetailPresenterHeader
```

Comment:
Are you sure this isn't used to present a resource name anywhere else? If so, I'm good with this rename.

Reply:
This isn't a rename, it's just switching out to the existing parent component of `ResourceNamePresenter`.